.github/workflows/dev-workflow.yml

```
@@ -232,4 +232,4 @@ jobs:
         with:
           base: main
           head: stg
-          token: ${{ secrets.GITHUB_TOKEN }}
+          token: ${{ secrets.GITHUB_TOKEN }}
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)